api/src/test/java/com/nicknathanjustin/streamercontracts/votes/VoteServiceImplTest.java

```diff
@@ -27,8 +25,8 @@
 
     private static final UUID PROPOSER_ID = UUID.randomUUID();
     private static final UUID STREAMER_ID = UUID.randomUUID();
-    // Subtract 1000ms from timestamp to address potential race conditions
-    private static final Timestamp SETTLE_CONTRACT_TIME_STAMP = new Timestamp(System.currentTimeMillis() - 1000);
+    // Subtract 10000ms from timestamp to address potential race conditions
```

Comment:
Did you see a race condition and failing tests with 1 second delay?

Reply:
Was getting a weird bug but I think it was caused by something else. Reverted